docs/application/web/guides/device/feedback.md

```diff
@@ -16,6 +16,28 @@ The main features of the Feedback API include:
 
 Each feedback pattern can have separate media files of sound and vibration type. For example, when receiving a new email message, a predefined sound and vibration feedback pattern can be played.
 
+## Prerequisites
+
+1. To make your application visible in the Tizen Store only for devices that support feedback vibration feature, since Tizen 5.0 the application must specify the following feature in the `config.xml` file:
```

Comment:
Questions:
Is that you can selecting make apps visible for devices that support feedback vibration feature... is supported from Tizen 5.0?
Or it was possible to make apps visible to specific devices, but from Tizen 5.0 to continue using that feature you must specify it in config.xml file?

Accordingly I can suggest a rephrase.